src/devices/bus/snes/st018.cpp

```diff
@@ -0,0 +1,161 @@
+// license:BSD-3-Clause
+// copyright-holders:Fabio Priuli
```

Comment:
If you did this then it's legit not an @etabeta78 attribution

Reply:
It's based on bus/snes/upd.cpp, that's why i unchanged this.